Notebook updates and addition of US-CRT config file

inkenbrandt opened the pull request:
- Update example notebooks and delete redundant license files
- Made modifications to the example notebooks to improve readability and code execution. These changes improve code organization and module imports, and enhance the use case guide for the users.  This includes dynamic references to the python files in the  `scripts` folder.
- Fixed deprecation issues with the pyproj function calls in the example notebooks
- Created a Config file for US-CRT, and fixed custom variable additions.

This has been tested using the notebooks.

JohnVolk commented:
Hey Paul, can you please share the full environment you are using? I'm having some issues with one of the example notebooks and I am not sure if it is due to different package versions or something else. Just to be clear, when you tested using the AmeriFlux data the resulting example plots in the notebook looked exactly as they did before the changes?

inkenbrandt commented:
I refactored the `calc_footprint_FFP_climatology.py` into a new class in `ffp.py`, so that is likely the issue.  I also added some basic tests.  The celery notebook links to the new class.
